cmd/krew/cmd/list.go

```diff
@@ -67,6 +69,8 @@ Remarks:
 		PreRunE: checkIndex,
 	}
 
+	listCmd.Flags().BoolVarP(overrideFlag, "override", "o", false, "override standard limited terminal output behavior")
```

Comment:
`override` does not say what it overrides and is thus not a good name. I think `--wide` is pretty common in k8s and should convey the workings quite well.

Reply:
I'll rename it right now

Reply:
Renamed to "--wide"/"-w" and added "--names-only"